Doctrine/EventSubscriber.php

```diff
@@ -64,7 +66,12 @@ public function persistObject(DataEvent $event)
         }
 
         $objectManager->persist($event->getData());
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
Great, I didn't know `finally` feature. But is it also executed on success, or only when an exception occurred ?